src/js/static.js

```diff
@@ -1,3 +1,5 @@
+const MAXIMUM_LIST_COUNT = /* istanbul ignore next */ _TEST_ ? 30 : 500;
```

Comment:
Set to a maximum of `500` in production. And `30` in Cypress.

Loading a list of `500` actions/flows in Cypress was making me nervous that it would cause issues. 

The largest list I could find in the tests was `20`, so I figured `30` was safe. But maybe we want to raise it to `50` or something a little higher to be sure other tests don't unknowingly reach that max in the future... 🤔

Reply:
not a bad idea.  if this number changes in the future, let's request it get put on the meta maybe?  I'm not 100% sure how we're going to get back that result or handle it, but for now I think your static option works well.